test/tasks_screen/bloc/tasks_event_test.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:lachies_life_planner/tasks_screen/bloc/task_event.dart';
+
+import '../../utils/mock_firestore_data.dart';
+
+void main() {
+  group('TasksEvent', () {
+    test('LoadTasksEvents can be compared with deep equality', () {
+      expect(LoadTasksEvent(), LoadTasksEvent());
+    });
+
+    test('TasksLoadedEvent can be compared with deep equality', () {
+      expect(TasksLoadedEvent(mockTaskList), TasksLoadedEvent(mockTaskList));
+      expect(TasksLoadedEvent(mockTaskList) == TasksLoadedEvent(mockTaskList + [mockTask]), false);
+    });
+
+    test('AddTaskEvent can be compared with deep equality', () {
+      expect(AddTaskEvent(mockTask), AddTaskEvent(mockTask));
+      expect(AddTaskEvent(mockTask) == AddTaskEvent(mockTask.copyWith(isComplete: true)), false);
+    });
+
+    test('UpdateTaskEvent can be compared with deep equality', () {
+      expect(UpdateTaskEvent(mockTask), UpdateTaskEvent(mockTask));
+      expect(UpdateTaskEvent(mockTask) == UpdateTaskEvent(mockTask.copyWith(isComplete: true)), false);
+    });
+
+    test('DeleteTaskEvent can be compared with deep equality', () {
+      expect(DeleteTaskEvent(mockTask), DeleteTaskEvent(mockTask));
+      expect(DeleteTaskEvent(mockTask) == DeleteTaskEvent(mockTask.copyWith(isComplete: true)), false);
+    });
+
+    test('DeleteAllTasksEvent can be compared with deep equality', () {
+      expect(DeleteAllTasksEvent(), DeleteAllTasksEvent());
+    });
```

Comment:
I know we want 100% test coverage, but these tests don't do much

Reply:
Yeah I deleted these.  Adding useless test files that I know are useless just to satisfy codecov sounds like bad practise.  Also there are sometimes things that we just can't test, and that is okay.  

Wait, I can ignore files from test coverage can't I?

Reply:
I can, but screw it.  95% test coverage is still cool